themes/xresources/theme.lua

```diff
@@ -59,11 +59,40 @@ theme.menu_width  = dpi(100)
 -- beautiful.variable in your rc.lua
 --theme.bg_widget = "#cc0000"
 
--- Recolor titlebar icons:
-theme = theme_assets.recolor_titlebar_normal(theme, theme.fg_normal)
-theme = theme_assets.recolor_titlebar_focus(theme, theme.fg_focus)
+-- Recolor Layout icons:
 theme = theme_assets.recolor_layout(theme, theme.fg_normal)
 
+-- Recolor titlebar icons:
+--
+local function darker(color_value, darker_n)
+    local result = "#"
+    for s in color_value:gmatch("[a-fA-F0-9][a-fA-F0-9]") do
```

Comment:
it can process only hex colors like (`#123456` and `#12345601`) but `xresources.get_theme()` return only them so it looks fine for me to have it locally here, if it will support any color input format it can be later on moved somewhere to `gears.color`